src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -69,14 +69,24 @@ public Window DynamoWindow
             dynamoMenu = dynamoView.titleBar.ChildOfType<Menu>();
             viewStartupParams = new ViewStartupParams(dynamoVM);
             DynamoSelection.Instance.Selection.CollectionChanged += OnSelectionCollectionChanged;
-
         }
 
         public void AddMenuItem(MenuBarType type, MenuItem menuItem, int index = -1)
         {
             AddItemToMenu(type, menuItem, index);
         }
 
+        /// <summary>
+        /// Adds the extension UI control to a new tab in the right side bar.
+        /// </summary>
+        /// <param name="viewExtension"></param>
```

Comment:
can you add some comments to the params?

Reply:
Added comments